int64 pose_flag
---
bool success